### DISCUSSION TITLE
Is there a way to avoid repeating `$todo.data.todo.id` etc?

### DISCUSSION THREAD

**Discussion by frederikhors** [urql-graphql/urql#1921] [Q&A]
Using the Svelte example in docs:

```svelte
<script>
  import { operationStore, query } from '@urql/svelte';

  const todos = operationStore(`
    query {
      todos {
        id
        title
      }
    }
  `);

  query(todos);
</script>

{#if $todos.fetching}
  <p>Loading...</p>
{:else if $todos.error}
  <p>Oh no... {$todos.error.message}</p>
{:else}
  <ul>
    {#each $todos.data.todos as todo}
      <li>{todo.title}</li>
    {/each}
  </ul>
{/if}
```

is there a way I can avoid to use `$todos.data.todos`?

Sometimes I have a big object and I'm repeating `$todo.data.todo.id`, `$todo.data.todo.name` etc.

Something like:

```ts
const [data, error, fetching] = operationStore()
```

**Comment by frederikhors** [comment]
I think I can use this but I don't know if is good enough (performances):

```ts
const todos = operationStore();

query(todos);

$: data = $todos.data?.todos
```

What do you think?

**Reply by JoviDeCroock** [reply]
Assigning to a variable should not really incur a performance cost 😅 you can even try [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)

**Reply by frederikhors** [reply]
Destructuring for array, but for object?
